### ISSUE TITLE
Gaussian, MoL or MoG instead of categorical

### ISSUE THREAD

**Issue by MichelPezzat** [openai/jukebox#266]
I got my attention in the future work mentioned in the paper. So, in order to distill the prior with IAF, is it necessary to estimate prior parameters based on a distribution other than categorical? Like Gaussian, MoL or so...This is based in parallel wavelet paper.
